servicetalk-transport-netty-internal/src/test/java/io/servicetalk/transport/netty/internal/SslCloseNotifyAlertClientHandlingTest.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright © 2020 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.transport.netty.internal;
+
+import io.servicetalk.concurrent.PublisherSource;
+import io.servicetalk.concurrent.api.test.FirstSteps;
+
+import org.junit.Test;
+
+import java.nio.channels.ClosedChannelException;
+
+import static io.servicetalk.concurrent.api.Processors.newPublisherProcessor;
+import static io.servicetalk.concurrent.api.SourceAdapters.fromSource;
+
+public class SslCloseNotifyAlertClientHandlingTest extends AbstractSslCloseNotifyAlertHandlingTest {
+
+    public SslCloseNotifyAlertClientHandlingTest() throws Exception {
+        super(true);
+    }
+
+    @Test
+    public void afterExchangeIdleConnection() {
+        sendRequest();
+        FirstSteps.create(conn.read())
```

Comment:
@Scottmitch as we discussed [here](https://github.com/apple/servicetalk/pull/1240#discussion_r543650104), I will submit a PR to rename it to `StepVerifiers`.
I will keep it without static import. In the context of `PublisherStepVerifierTest`, `create(from("foo", "bar"))` looks ok. But when the test is related to some business logic, things like `create(conn.read())` are a bit confusing. Not clear what are we creating. Something like `Verifiers.stepVerifier(...)` might be a better option.

Reply:
Renaming: #1286

Reply:
I'm fine with a more descriptive method name (e.g. `Verifiers.stepVerifier`). now is the time before we release 👍  feel free to open a PR.

Reply:
Verifiers: #1290